src/main/java/com/artipie/rpm/meta/XmlMetaJoin.java

```diff
@@ -27,20 +27,11 @@
 import java.io.BufferedReader;
 import java.io.BufferedWriter;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.OutputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.StandardCopyOption;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
-import javax.xml.stream.XMLEventFactory;
-import javax.xml.stream.XMLEventReader;
-import javax.xml.stream.XMLEventWriter;
-import javax.xml.stream.XMLInputFactory;
-import javax.xml.stream.XMLOutputFactory;
-import javax.xml.stream.XMLStreamException;
-import javax.xml.stream.events.XMLEvent;
 
 /**
  * Joins two meta xml-files.
```

Comment:
@olenagerasimova Seems that we already have another class which joins two xml files, and do it better. So why keep this old implementation?

Reply:
@paulodamaso old implementation (with xml-streams) was moved to test scope and now is used in `XmlMetaJoinITCase` to compare resulting files.

Reply:
@olenagerasimova Understood